code/game/objects/structures/tables_racks.dm

```diff
@@ -175,6 +175,7 @@
 		extra_wound = 20
 	banged_limb?.receive_damage(30, wound_bonus = extra_wound)
 	pushed_mob.apply_damage(60, STAMINA)
+	pushed_mob.SetSleeping(300) // SKYRAT EDIT
```

Comment:
30 seconds of sleeping on top of all of that, for a move that takes about 10 seconds to do, probably even less? And that you can probably repeat once they're asleep? Sounds a bit overly busted.